data/campaigns/tutorial/scenarios/02_Tutorial_part_2.cfg

```diff
@@ -869,6 +869,14 @@ A full list of abilities and weapons specials, along with traits, may be found i
         [/filter_condition]
 
         {TALK_ABOUT Thrag ( _ "You are close to killing their leader! The unit who finishes him will gain 16 experience points because he is second level. Choose your attacking unit carefully!")}
+
+        [message]
+            speaker=student
+            message= _ "Should I choose a unit that will level up?"
+            message_female= _ "female^Should I choose a unit that will level up?"
+        [/message]
+
+        {TALK_ABOUT Thrag ( _ "No, I'd recommend choosing one that needs 17-24 experience points to advance, so that they'll be about to level. Units are fully healed when they advance a level, and all units are fully healed between scenarios. Also, until they level, they'll cost less gold per turn for upkeep.")}
```

Comment:
This is a good rule of thumb in general, but I'm not sure it's good advice in this specific situation.

At this point, the player likely has no lvl2 units at all. A unit that's near advancement could become the player's first Captain, Druid, Sorceress, or Marksman, all of which are unusually better than their lvl1 counterparts due to their additional abilities or weapon specials. It's not a hard and fast rule, but it might be preferable to advance the first Captain/Druid/etc now than to start the next scenario (if there were one) with two lvl1 units near advancement.

Admittedly, this is a bit of an exceptional situation (it wouldn't apply in the middle of a campaign, nor when playing orcs or dwarves).

Reply:
Thanks, I've pushed a new version for this. It's becoming a bit of a wall-of-text, but I think it's not too much.

Reply:
Thanks, I like it very much :+1:

Some minor suggestions:

1. "is good" sounds a bit... bland. How about, say, ending the sentence before "and" and adding "Having these units already levelled on the recall list can be invaluable" as a final sentence?

2. "can stop a hard-to-hit enemy unit from blocking your path" sounds like it can prevent an enemy unit from entering a hex in the player's path. How about "are good for dislodging fortified enemy units" for example.

3. "for upkeep" -> "in upkeep"

4. In "and if you are facing ... then her ability ..." how about reversing the order: "and her ability ... is invaluable if you are facing ...".

Reply:
Many thanks. I didn't want to use "invaluable" twice, and having swapping one of them to "priceless" didn't feel right either. I've gone for "... on your recall list can make a huge difference in the next battle.", although it's an even longer wall-of-text.

Reply:
Absolutely agreed on not using "invaluable" twice; I hoped you'd find a better way to phrase that part.

My only comment on the current revision is that "a Sorceress or Marksman" is unbalanced; either add an indefinite article to "Marksman", or change to plural. Sorry for not spotting that previously.

I think the end of the tutorial is a good place to hint at the depth of strategic considerations, though we should still allow players to skip through it. About the wall of text, I don't see a way to shorten the description of the Captain/Druid/etc, but we could split the last two sentences of the "I'd normally..." line to a new message by `speaker=narrator`, or delete the penultimate sentence entirely (it's documented in the help, and the player will discover it soon enough anyway).

Reply:
I've added some suggested changes below.

Reply:
@stevecotton Have you seen @nemaara's suggestions? Do you agree with them?